cmd/generate/codeowners/output.go

```diff
@@ -3,23 +3,37 @@ package codeowners
 import (
 	"fmt"
 	"os"
+	"path/filepath"
 	"regexp"
 	"sort"
 	"strings"
 
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
 	"github.com/open-sauced/pizza-cli/pkg/config"
 )
 
-func generateOutputFile(fileStats FileStats, outputPath string, ownersStyleFile bool, config *config.Spec) error {
+func generateOutputFile(fileStats FileStats, outputPath string, opts *Options, cmd *cobra.Command) error {
 	// Open the file for writing
 	file, err := os.Create(outputPath)
 	if err != nil {
 		return fmt.Errorf("error creating %s file: %w", outputPath, err)
 	}
 	defer file.Close()
+	var flags []string
+
+	cmd.Flags().Visit(func(f *pflag.Flag) {
+		flags = append(flags, fmt.Sprintf("--%s %v", f.Name, f.Value.String()))
```

Comment:
Since `f.Value.String()` returns a string, let's use the `%s` string format placeholder instead of the typeless "value" placeholder (which can, at times, cause some strange behavior)

```suggestion
		flags = append(flags, fmt.Sprintf("--%s %s", f.Name, f.Value.String()))
```